plugins/woocommerce/tests/api-core-tests/tests/products/product-list.test.js

```diff
@@ -3195,8 +3195,6 @@ test.describe( 'Products API tests: List All Products', () => {
 				} );
 			} );
 
-			// This case will remain skipped until orderby include is fixed.
-			// See: https://github.com/woocommerce/woocommerce/issues/30354#issuecomment-925955099.
```

Comment:
This test wasn't being skipped anymore, but the comment was still here.  So I just removed it to clean things up.